openedx/core/djangoapps/edx_global_analytics/tests/test_utils.py

```diff
@@ -0,0 +1,157 @@
+"""
+Tests for edx_global_analytics application helper functions aka utils.
+"""
+
+import datetime
+
+from django.test import TestCase
+from django.utils import timezone
+
+from django.db.models import Q
+from django_countries.fields import Country
+
+from student.models import UserProfile
+from student.tests.factories import UserFactory
+
+from ..utils import fetch_instance_information, cache_instance_data
+
+
+class TestStudentsAmountPerParticularPeriod(TestCase):
+    """
+    Tests cover all methods, that have a deal with statistics calculation.
+    """
+    @staticmethod
+    def active_students_amount_default_data():
```

Comment:
It's considered a bad practice to plant a side-effect into a function. See https://stackoverflow.com/a/766736/231412